android/app/src/main/res/xml/authenticator.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<account-authenticator xmlns:android="http://schemas.android.com/apk/res/android"
+    android:accountType="org.smartregister.fhircore.app"
```

Comment:
We've previously had issues when reusing this functionality on different apps since it requires redeclaring of this file so as to override this value. We can move this value to a `strings.xml` and add the property `translatable="false"` to the string declaration. 

What do you think?